src/dev/impl/DevToys/Views/Tools/Text/MarkdownPreview/MarkdownPreviewToolPage.xaml.cs

```diff
@@ -65,5 +69,25 @@ public void Receive(NavigateToMarkdownPreviewHtmlMessage message)
             Arguments.NotNull(message, nameof(message));
             PreviewWebView.NavigateToString(message.Html);
         }
+
+        private async void CopyButton_Click(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                var data = new DataPackage
+                {
+                    RequestedOperation = DataPackageOperation.Copy
+                };
+                string markdownBodyEl = "document.getElementsByClassName('markdown-body')[0].innerHTML";
```

Comment:
I have to admit that this whole method is mostly an exact copy of similar functionality I found in the `CodeEditor` control class except for the strategy of extracting the HTML body from the webview. Speaking of which, is there a better way to access it? (somewhat related to my second point in my [comment](https://github.com/veler/DevToys/pull/260/files#r792285172) above.)

Reply:
See my previous comment :)